src/main/resources/templates/mdBenefitsFlow/choosePrograms.html

```diff
@@ -14,18 +14,20 @@
             th:replace="~{fragments/form :: form(action=${formAction}, content=~{::formContent})}">
           <th:block th:ref="formContent">
             <div class="form-card__content">
+              <h2 th:text="#{choose-programs.check-all-that-apply}"></h2>
               <th:block th:replace="~{fragments/inputs/checkboxFieldset ::
                   checkboxFieldset(inputName='programs',
-                  label=#{choose-programs.check-all-that-apply},
+                  ariaLabel=#{choose-programs.header},
```

Comment:
@bseeger - do we have a best practice around what to include in the ariaLabel?

Reply:
no, I was wondering that as well. Might be worth checking in with Devon about this.

Reply:
It should be the ID of the element you want to label the input. It's meant for screens where there is only one input, and the header is technically the label for that input. In most cases it will be 'header' since that's the ID assigned to headers in the FFB.